### DISCUSSION TITLE
OS-wide monospaced font not being recognized

### DISCUSSION THREAD

**Discussion by GlenHenshaw** [blinksh/blink#1700] [Support]
### Checklist

- [x ] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [ x] Read the [docs](http://blink.sh/docs/)
- [ x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

IPadOS 16.3, blink version 16.0.4.644

### Describe the bug

I'm trying to use a commercial monospaced font, Berkeley Mono (https://berkeleygraphics.com/typefaces/berkeley-mono). I know it is installed on the system, because Pages recognizes it. But it does not show up in the font selection list for blink.

Specifically, I am currently using the trial typeface, which includes a small subset of all the available glyphs.

Berkeley mono trial should appear in the blink font selection list.

**Comment by carloscabanero** [comment]
Blink is showing fonts that identify to the system as monospaced. We still recommend to import the font in Blink instead of in the system: https://github.com/blinksh/blink/discussions/1540

**Comment by GlenHenshaw** [comment]
OK, but Berkeley Mono *does* identify to the system as a monospaced font. 

I'll try the direct import, but it's a lot more work - I don't have a .css file, just .otf and .ttf files, so I'll have to make one, then I'll have to set up a repo blink can pull from.
